src/agent/src/image.rs

```diff
@@ -66,26 +75,41 @@ impl ImageService {
         if !guest_pause_bundle.exists() {
             bail!("Pause image not present in rootfs");
         }
+        let guest_pause_config = guest_pause_bundle.join(CONFIG_JSON);
 
         info!(sl(), "use guest pause image cid {:?}", cid);
         let pause_bundle = Path::new(CONTAINER_BASE).join(cid).join(target_subpath);
         let pause_rootfs = pause_bundle.join("rootfs");
         fs::create_dir_all(&pause_rootfs)?;
-
-        let copy_if_not_exists = |src: &Path, dst: &Path| -> Result<()> {
-            if !dst.exists() {
-                info!(sl(), "copying file {src:?} to {dst:?}");
-                fs::copy(src, dst)?;
-            }
-            Ok(())
-        };
-        copy_if_not_exists(
-            &guest_pause_bundle.join(CONFIG_JSON),
-            &pause_bundle.join(CONFIG_JSON),
-        )?;
+        info!(sl(), "pause_rootfs {:?}", pause_rootfs);
+        let image_oci = oci::Spec::load(guest_pause_config.to_str().ok_or_else(|| {
+            anyhow!(
+                "Invalid container image OCI config path {:?}",
```

Comment:
I suggest re-phrase this message like: "Failed to load the guest pause image config from {:?}"

Reply:
Fixed.